package.json

```diff
@@ -78,11 +78,11 @@
     "xmlbuilder": "^2.6.1"
   },
   "devDependencies": {
-    "browserify": "^9.0.3",
+    "browserify": "^10.1.3",
```

Comment:
We don't have tests for the browser version. Anything worrisome in that major version bump?